try:
    1+1
except ValueError as ex:
    raise ex
finally:
    2 + 2